api/gateway-operator/v2alpha1/controlplane_types.go

```diff
@@ -147,6 +147,14 @@ type ControlPlaneDataPlaneSync struct {
 	// +optional
 	// +kubebuilder:default=false
 	EnableReverse *bool `json:"enableReverse,omitempty"`
+	// Interval is the interval between two rounds of syncing Kong configuration with dataplanes.
+	//
+	// +optional
+	Interval *metav1.Duration `json:"interval,omitempty"`
```

Comment:
REVIEW: set a minimal value and a default value for the interval?

Reply:
I think that the same value as in KIC should be used